src/api/definitions/asset.ts

```diff
@@ -2,6 +2,9 @@ import { Blockchain } from './blockchain';
 
 export const AssetUrl = { get: `asset?blockchains=${Object.values(Blockchain).join(',')}` };
 
+export const AddAssetToMetaMaskDesc =
+  'Click on the MetaMask symbol in order to add this asset in your portfolio overview of your MetaMask or copy the address to add it manually.';
```

Comment:
This is probably not the right place for this description ... but I'm sure @Krysh90 will take care of this 😉